diff -urN whois-4.7.17.org/tld_serv_list whois-4.7.17/tld_serv_list
--- whois-4.7.17.org/tld_serv_list	2006-07-19 18:57:55.000000000 +0200
+++ whois-4.7.17/tld_serv_list	2006-09-28 18:30:09.000000000 +0200
@@ -37,7 +37,7 @@
 .coop	whois.nic.coop
 .info	whois.afilias.info
 .jobs	whois.jobs
-.mobi	WEB http://pc.mtld.mobi/whois/
+.mobi	whois.dotmobiregistry.net
 .museum	whois.museum
 .name	whois.nic.name
 .pro	whois.registrypro.pro
