--- whois-4.6.9/Makefile.orig	2003-12-01 18:03:18.000000000 +0000
+++ whois-4.6.9/Makefile	2003-12-02 15:48:05.000000000 +0000
@@ -1,4 +1,6 @@
-prefix ?= /usr/local
+prefix ?=/usr
+bindir=$(prefix)/bin
+mandir=/usr/share/man
 
 OPTS=-O2
 
@@ -38,12 +40,12 @@
 	perl make_tld_serv.pl < tld_serv_list > tld_serv.h
 
 install: whois
-	install -m 0755 whois $(BASEDIR)$(prefix)/bin/
-	strip $(BASEDIR)$(prefix)/bin/whois
-	install -m 0644 whois.1 $(BASEDIR)$(prefix)/share/man/man1/
+	install -d $(DESTDIR)$(bindir)
+	install -d $(DESTDIR)$(mandir)/man1
+	install -s -m 0755 whois $(DESTDIR)$(bindir)
+	install -m 0644 whois.1 $(DESTDIR)$(mandir)/man1
 	cd po && $(MAKE) $@
 
-
 distclean: clean
 	rm -f po/whois.pot
 

diff -Nru whois-4.6.9/po/Makefile whois-4.6.9.new/po/Makefile
--- whois-4.6.9/po/Makefile	2003-06-15 16:46:16.000000000 +0000
+++ whois-4.6.9.new/po/Makefile	2003-12-02 15:55:37.000000000 +0000
@@ -1,5 +1,5 @@
 INSTALL= /usr/bin/install -c
-INSTALLNLSDIR=$(BASEDIR)/usr/share/locale
+INSTALLNLSDIR=$(DESTDIR)/usr/share/locale
 
 PACKAGE = whois
 
