Extended malloc attributes are available since gcc 11.
--- whois/utils.h.orig	2023-10-08 22:58:33.000000000 +0200
+++ whois/utils.h	2023-10-21 17:56:23.225691505 +0200
@@ -17,7 +17,11 @@
 /* Portability macros */
 #ifdef __GNUC__
 # define NORETURN __attribute__((noreturn))
+#  if __GNUC__ >= 11
 # define MALLOC_FREE __attribute__((malloc(free)))
+#  else
+#   define MALLOC_FREE
+#  endif
 # define NONNULL __attribute__((returns_nonnull))
 #else
 # define NORETURN
