diff -Nru whois-4.4.4/Makefile whois-4.4.4.new/Makefile
--- whois-4.4.4/Makefile	Wed Dec 15 19:45:57 1999
+++ whois-4.4.4.new/Makefile	Wed Dec 15 19:45:28 1999
@@ -1,4 +1,5 @@
-prefix=/usr/local
+prefix=/usr
+mandir=/usr/share/man
 
 OPTS=-O2
 
@@ -23,10 +24,10 @@
 	perl make_tld_serv.pl < tld_serv_list > tld_serv.h
 
 install: whois
-	install --strip -m 0755 whois $(BASEDIR)$(prefix)/bin/
-	install --strip -m 0644 whois.1 $(BASEDIR)$(prefix)/man/man1/
+	install -d $(DESTDIR)$(prefix)/bin $(DESTDIR)$(mandir)/man1
+	install --strip -m 0755 whois $(DESTDIR)$(prefix)/bin/
+	install -m 0644 whois.1 $(DESTDIR)$(mandir)/man1/
 	cd po && $(MAKE) $@
-
 
 distclean: clean
 	rm -f po/whois.pot
diff -Nru whois-4.4.4/po/Makefile whois-4.4.4.new/po/Makefile
--- whois-4.4.4/po/Makefile	Wed Dec 15 19:45:57 1999
+++ whois-4.4.4.new/po/Makefile	Wed Dec 15 19:43:51 1999
@@ -1,5 +1,5 @@
 INSTALL= /usr/bin/install -c
-INSTALLNLSDIR=$(BASEDIR)/usr/share/locale
+INSTALLNLSDIR=$(DESTDIR)/usr/share/locale
 
 PACKAGE = whois
 
