diff -urNp -x '*.orig' whois.org/whois.c whois/whois.c
--- whois.org/whois.c	2019-12-31 12:14:30.000000000 +0100
+++ whois/whois.c	2021-11-16 22:11:57.631795697 +0100
@@ -1001,8 +1001,12 @@ int openconn(const char *server, const c
     hints.ai_flags = AI_ADDRCONFIG;
     hints.ai_flags |= AI_IDN;
 
-    if ((err = getaddrinfo(server, port ? port : "nicname", &hints, &res))
-	    != 0) {
+    err = getaddrinfo(server, port ? port : "nicname", &hints, &res);
+    if (err == EAI_BADFLAGS) {
+	    hints.ai_flags &= ~AI_IDN;
+	    err = getaddrinfo(server, port ? port : "nicname", &hints, &res);
+    }
+    if (err != 0) {
 	if (err == EAI_SYSTEM)
 	    err_sys("getaddrinfo(%s)", server);
 	else
